ios/objc/testApp/objc/TestApp/TestApp/utils/ImagePredictor.mm

```diff
@@ -0,0 +1,69 @@
+#import "ImagePredictor.h"
+#import <torch/script.h>
+#import <PytorchObjC/PytorchObjC.h>
+#import <vector>
+#import "UIImage+Utils.h"
+
+#define IMG_W 224
+#define IMG_H 224
+#define IMG_C 3
+
+@implementation ImagePredictor {
+    TorchModule* _module;
+    NSArray* _labels;
+}
+
+- (instancetype)initWithModelPath:(NSString* )modelPath {
+    self = [super init];
+    if (self) {
+        _module = [TorchModule loadTorchscriptModel:modelPath];
+        NSError* err;
+        NSString* str = [NSString stringWithContentsOfFile:[[NSBundle mainBundle] pathForResource:@"synset_words" ofType:@"txt"]
+                                                  encoding:NSUTF8StringEncoding
+                                                     error:&err];
+        _labels = [str componentsSeparatedByCharactersInSet:[NSCharacterSet newlineCharacterSet]];
+    }
+    return self;
+}
+
+- (void)predict:(UIImage* )image
+Completion:(void(^__nullable)(NSArray<NSDictionary* >* sortedResults))completion {
```

Comment:
Normally, the completion shouldn't be nullable here, since there's no point of invoking this method and not being interested in the result. Can assert the `completion` isn't `nil`.